arches_her/pkg/graphs/resource_models/Activity.json

```diff
@@ -14538,6 +14545,11 @@
                     "alias": "shine_form",
                     "config": {
                         "options": [
+                            {
+                                "id": "e8d04473-da05-4cda-95f4-ebfd0ff28122",
+                                "selected": false,
+                                "text": "N/A"
+                            },
                             {
                                 "id": "3620365b-6294-4356-93c0-52f53a2e4a49",
                                 "selected": false,
```

Comment:
@SDScandrettKint this still seems to be a concept node with domain node config

Reply:
Apologies, completely missed the Activity model.